packages/clima_ui/lib/utilities/failure_snack_bar.dart

```diff
@@ -38,7 +38,7 @@ void showFailureSnackBar(
       action: onRetry != null
           ? SnackBarAction(
               label: 'Retry',
-              textColor: Theme.of(context).colorScheme.secondary,
+              textColor: Theme.of(context).snackBarTheme.actionTextColor,
```

Comment:
Instead of doing this, you should modify the `contentTextStyle` of `snackBarTheme` in each theme, though it already has a different color. Why is that?

Reply:
I don't get what you mean.